translations/back_cover.tex/de.po

```diff
@@ -82,3 +110,19 @@ msgid ""
 "\\svg{building_materials} \\svgeven[7]{arrow_right_gray} 1 "
 "\\svg{valuablegreater}} & \\lightcell{--} \\\\ \\end{tabularx} }"
 msgstr ""
+"\\hommtable[]{14}{ \\centering \\textbf{\\pagetarget{Trade Table}{Übersicht "
+"Handelskurse}}\\\\ \\bigskip \\begin{tabularx}{\\linewidth}{XXXX} "
+"\\darkcell{\\color{white}Verkaufe/ Erhalte} & \\darkcell{… und erhalte "
+"\\svg{gold}} & \\darkcell{… und erhalte \\svg{valuablegreater}} & "
+"\\darkcell{… und erhalte \\svg{building_materials}} \\\\ \\darkcell{Ich "
```

Comment:
```suggestion
"\\darkcell{\\color{white}Verkaufe/ Erhalte} & \\darkcell{... und erhalte "
"\\svg{gold}} & \\darkcell{... und erhalte \\svg{valuablegreater}} & "
"\\darkcell{... und erhalte \\svg{building_materials}} \\\\ \\darkcell{Ich "
```

Reply:
What's wrong with …?

Reply:
Consistency. The table looks very asymmetric now.